plugins/callback/logstash.py

```diff
@@ -18,40 +18,85 @@
     options:
       server:
         description: Address of the Logstash server
-        env:
-          - name: LOGSTASH_SERVER
+        version_added: 1.0.0
         ini:
           - section: callback_logstash
             key: server
-            version_added: 1.0.0
+        env:
+          - name: LOGSTASH_SERVER
         default: localhost
       port:
         description: Port on which logstash is listening
-        env:
-            - name: LOGSTASH_PORT
+        version_added: 1.0.0
         ini:
           - section: callback_logstash
             key: port
-            version_added: 1.0.0
+        env:
+          - name: LOGSTASH_PORT
         default: 5000
+      pre_command:
+        description: Executes command before run and result put to ansible_pre_command_output field.
+        version_added: "1.3.0"
```

Comment:
Next release will be 2.0.0, so you need to update the `version_added` for the new options:
```suggestion
        version_added: "2.0.0"
```